zedenv/cli/mount.py

```diff
@@ -106,6 +107,20 @@ def zedenv_mount(boot_environment: str, mountpoint: Optional[str], verbose: bool
         }, verbose)
         mount_children(child_datasets, mountpoint, verbose)
 
+    if check_bpool:
+        # If a separate ZFS boot pool is used
+        # mount the corresponding dataset to `{mountpoint}/boot`
+        if zedenv.lib.be.extra_bpool():
+            boot_environment_boot = zedenv.lib.be.root('/boot')
+            be_boot_requested = f"{boot_environment_boot}/zedenv-{boot_environment}"
+            try:
+                zedenv.lib.system.zfs_manual_mount(be_boot_requested, f"{mountpoint}/boot")
```

Comment:
This fails if the directory `{mountpoint}/boot` does not exists. Should we create it (`mkdir -p ...`)?

Reply:
I would think it would always exist, since it's also where were searching for kernels. What do you think? I guess there shouldn't be a downside to creating it...